### DISCUSSION TITLE
ilogtail_config.json 配置 max_bytes_per_sec 和 cpu_usage_limit 不生效？

### DISCUSSION THREAD

**Discussion by dongwang123** [alibaba/ilogtail#706] [Help] [answered]
使用 ilogtail 收集业务日志（270M/s），ilogtail_config.json 配置如下：
<img width="672" alt="image" src="https://user-images.githubusercontent.com/26740892/224956350-e7a0cb64-4bf7-4e46-a96d-6afde38a344c.png">

实际 ilogtail 的 CPU 使用率如下：
<img width="771" alt="image" src="https://user-images.githubusercontent.com/26740892/224956708-1598d6f3-0c97-4ead-a7ae-7c98fe48a7cc.png">

实际发送 kafka 的流量如下：
<img width="905" alt="image" src="https://user-images.githubusercontent.com/26740892/224956862-830e38b1-6d79-402e-9fc0-bfeb4b94b766.png">

我理解 max_bytes_per_sec 配置为 500000，最大发送字节数就是 500000，看起来好像不是这样，是我理解错了吗？
另外 cpu_usage_limit 设置为2 ，好像也和预期结果不符

**Comment by messixukejia** [comment, marked as answer]
cpu是跟数据量规格有关的，之前测试结果供参考 https://mp.weixin.qq.com/s/8mCVk3gvXPOijTlcRjUR_w

你们真实单机这么大的数据量，还是只在做压测？ 具体明天双周会可以线上交流下

如果使用ilogtail请在登记下信息，方便我们更好的支持。https://github.com/alibaba/ilogtail/issues/693

**Reply by dongwang123** [reply]
目前只在做压测
